packages/airnode-validator/exampleSpecs/secrets.env

```diff
@@ -0,0 +1,4 @@
+gateway_key=012345678901234567890123456789
+airnode_rrp=0x534DfA0d242e7fDCFD096C2B1a3D70F172008778
+cloud_provider=aws
+node_version=0.2.0
```

Comment:
Can you name the variables as we usually do, all uppercase?

Reply:
Renamed in 7c90b7f